modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -24,11 +24,11 @@ The *Display name* and *Description* fields provide supplementary information an
 
 . Click *Create*.
 
-. Apply the the {Istio} injection label to the `bookinfo` namespace by entering the following command at the command line:
+. Apply the {Istio} injection label to the `bookinfo` namespace by entering the following command at the CLI:
 +
 [source,terminal]
 ----
-$ oc label namespace bookinfo istio.io/rev=<revision-name>
+$ oc label namespace default istio-injection=enabled
```

Comment:
```suggestion
$ oc label namespace bookinfo istio-injection=enabled
```

Reply:
For the "oc label namespace bookinfo istio.io/rev=<revision-name>" vs. "oc label namespace bookinfo istio-injection=enabled" second command will only work if the revision is named "default" in all other cases we need first command. This should be documented to avoid confusion.

Reply:
Yeah, we can add the following note similar to what we captured in [Sail Operator docs](https://github.com/istio-ecosystem/sail-operator/blob/e1fcbdb9517a87c42df6d82a7e7c38450344534c/docs/README.md?plain=1#L308).

Note: if the `Istio` resource name is other than `default`, you need to set the `istio.io/rev` label to the name of the `Istio` resource instead of adding the `istio-injection=enabled` label.